app/views/vm_common/_live_migrate.html.haml

```diff
@@ -1,67 +1,12 @@
-%form#form_div{:name           => "angularForm",
-               'ng-controller' => "vmCloudLiveMigrateFormController as vm",
-               "miq-form"      => true,
-               "model"         => "vm.vmCloudModel",
-               "model-copy"    => 'vm.modelCopy',
-               'form-changed'  => ''}
-  = render :partial => "layouts/flash_msg"
-  %h3
-    = _('Migrate Instance')
-  .form-horizontal
-    .form-group
-      %label.col-md-2.control-label
-        = _('Auto-select Host?')
-      .col-md-8
-        %input{:type         => "checkbox",
-               :name         => "auto_select_host",
-               :required     => true,
-               'ng-model'    => "vm.vmCloudModel.auto_select_host",
-               'ng-disabled' => "vm.hosts.length == 0"}
-
-        %input{:type         => "hidden",
-               :name         => "auto_select_host",
-               :value        => "on",
-               'ng-if'       => "vm.hosts.length == 0"}
-    .form-group
-      %label.col-md-2.control-label
-        = _('Block Migration')
-      .col-md-8
-        %input{:type        => "checkbox",
-               :name        => "block_migration",
-               :required    => true,
-               'ng-model'   => "vm.vmCloudModel.block_migration"}
-    .form-group
-      %label.col-md-2.control-label
-        = _('Disk Over Commit')
-      .col-md-8
-        %input{:type        => "checkbox",
-               :name        => "disk_over_commit",
-               'ng-model'   => "vm.vmCloudModel.disk_over_commit",
-               :required    => true}
-  #live-migrate-select-destination-host{'ng-hide' => 'vm.vmCloudModel.auto_select_host'}
-    %h3
-      = _('Select Destination Host')
-    .form-horizontal
-      .form-group
-        %label.col-md-2.control-label
-          = _('Destination Host')
-        .col-md-8
-          %select{:name      => 'destination_host_id',
-                  'ng-model' => 'vm.vmCloudModel.host',
-                  'ng-options' => 'host.name as host.name for host in vm.hosts track by host.id'}
-
-  = render :partial => 'layouts/angular/custom_form_buttons_controller_as', :locals => {:button_label => _('Submit'), :button_click => 'vm.submitClicked()'}
-  .clearfix
+%h3
+  = _('Migrate Instance')
+= react('LiveMigrateForm', {:recordId => @live_migrate_items.length == 1 ? @live_migrate_items[0].id : ''})
```

Comment:
I think it is better to do this inside `LiveMigrateForm` and just pass the props like
`{:items => @live_migrate_items}`